controllers/managedocs_controller.go

```diff
@@ -104,6 +105,8 @@ const (
 	providerDeploymentType             = "provider"
 	rhobsRemoteWriteConfigIdSecretKey  = "prom-remote-write-config-id"
 	rhobsRemoteWriteConfigSecretName   = "prom-remote-write-config-secret"
+	OVNNetworkType                     = "ovn"
+	SDNNetworkType                     = "sdn"
```

Comment:
- might be a bit verbose and at the same time I think it's better to have new type only for network 🤔
- cause we have only two possible values and better force them 

``` go
type NetworkType string
// ...
const (
  NetworkOVN NetworkType = "ovn"
  NetworkSDN NetworkType = "sdn"
)
// ...
type ManagedOCSReconciler struct {
// ...
  Network NetworkType
// ...
}
```

Reply:
The name NetworkType for the enum is misleading, we use the same network in both cases, it is just the CNI implementation that is different. I would suggest to use the name `NetworkInterfaceType` instead

In that case the values will be of the form `NetwortIntefaceOVN` and `NetworkInterfaceSDN`